maven/pom.xml

```diff
@@ -21,8 +21,13 @@
     <dependency>
       <groupId>com.google.jsinterop</groupId>
       <artifactId>jsinterop-annotations</artifactId>
-      <version>1.0.2</version>
+      <version>2.0.0</version>
     </dependency>
+    <dependency>
+      <groupId>com.vertispan.j2cl</groupId>
+      <artifactId>gwt-internal-annotations</artifactId>
+      <version>0.4-SNAPSHOT</version>
+    </dependency>  
   </dependencies>
```

Comment:
Annotations in this dep, are referenced and missing, this fix makes for a correctly packaged artifact.